chapters/en/slides/module6/module6_01.Rmd

```diff
@@ -0,0 +1,340 @@
+---
+params:
+  dynamictitle: "module6_01"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+<!-- #region -->
+# Telling stories with visualizations
+
+Notes:
+In this module we will see how we can add a narrative to our visualizations.
+Together,
+our narrative and our visualizations
+will be telling a data story.
+This does not mean
+that we will be making up fairy tales about our data
+but that we will guide the reader
+through the most important insights that we have found.
+
+---
+
+## How to tell a story
+
+- Start with an have a hook
+- explain
+- Leave the reader curios about the next fig
+
+Notes:
+An effective story often peaks the readers' interest up front,
+and makes them interested in reading further.
+For example, we could start with a visually striking figure,
+an important problem that begs for a solution,
+or reveal data that challenges the reader's previous knowledge.
+
+When making a data story exciting
+it is important to still remain fact-based
+to avoid that our story comes across as the headlines
+of a clickbait article.
+
+When telling a story,
+it is also important to tailor it to the audience and context
+to ensure that it is of the right technical difficulty and detail.
+
+To practice storytelling,
+we will show an [example based on articles on deforestation
+from Our World in Data](https://ourworldindata.org/forests-and-deforestation).
+
+
+---
+
+## A tail of global deforestation
+<!-- #endregion -->
+
+```{python}
+import pandas as pd
+import altair as alt
+
+# TODO save wrangled data and change to area plot shoing both tropical and temperate deforestation
+forest_loss = pd.read_csv('data/global-forest-loss.csv', parse_dates=['decade'])
+forest_loss['cumulative_hectar_lost_millions'] = forest_loss['hectar_lost_millions'].cumsum()
+
+alt.Chart(forest_loss).mark_line().encode(
+    x='decade',
+    y=alt.Y('cumulative_hectar_lost_millions'))
+```
+
+Notes:
+As we can see in this slide
+the total forest loss during the last 300 years
+is about 1.5 billion hectare.
+That's almost twice the area of Australia!
+
+To put this into a historical perspective
+the world has lost 1/6 of its forest in the last 100 years,
+which is as much as in the previous 9000 years combined.
+
+Although we can see that global deforestation
+has been slowing down since the 80s,
+it is still progressing at an unsustainable rate.
+
+Of the deforestation today,
+95% occurs in tropical regions,
+which is particularly severe
+since old rain forests are not easily replaced after they are cut down.
+
+---
+
+## Half of the tropical deforestation is localized to only two countries
+
+```{python}
```

Comment:
The main advantage I see with having them in the same plot is that they are directly comparable and we can see that Brazil and Indonesia has deforestation comparable to entire continents. I thought about removing mexico and India to streamline a bit. I might dothat. I will at least mentioning something about this in the text, but I will label this as "nice-to-have", let me know if you think it is critical.

Reply:
Ok I fixed this to remove the countries I mentioned at least. It would be nice with all countries for each continent but we don't have that data easily accessible.